static_src/components/usage_and_limits.jsx

```diff
@@ -217,9 +204,21 @@ export default class UsageAndLimits extends React.Component {
     );
   }
 
-  _onSubmit(e) {
-    e.preventDefault();
-    appActions.updateApp(this.props.app.guid, this.state.partialApp);
+  _onSubmit(errors, form) {
+    if (errors.length) {
+      console.warn({ errors }); // eslint-disable-line no-console
```

Comment:
No reason, _onSubmit taking an error is left over from the old `onValid` handler. By default, on submit, the `Form` component will display a message if there are errors. This behavior already existed and I didn't want to change it, although I think it needs some rethinking. Here's what it looks like. I'll remove the `console.warn` for now.

![peek 2017-03-01 16-55](https://cloud.githubusercontent.com/assets/509703/23488197/f369b57a-fe9f-11e6-9605-30c48fc0620a.gif)